tools/hangar/src/package.setup.ts

```diff
@@ -66,6 +66,11 @@ export default async function () {
     />/,
     `Install contains unexpected script hook: \n${installResult.stdout}`
   );
+  nodeAssert.doesNotMatch(
```

Comment:
(nit) Any reason this can't just be `assert`?

Reply:
So originally when I tried `assert` I remember there being some issue where it was using the `vitest` `assert` global instead. I figured vitest was hijacking something.

I must have just made a mistake before because it works fine now, thanks for bringing it up.